packages/webcomponents/src/components/business-details/business-details.tsx

```diff
@@ -86,12 +87,20 @@ export class BusinessDetails {
     return (
       <Host>
         <generic-info-details business={this.business} />
-        <legal-address-details legalAddress={this.business.legal_address} />
-        <representative-details
-          representative={this.business.representative}
-        />
-        <owner-details owners={this.business.owners} />
-        <additional-questions-details additionalQuestions={this.business.additional_questions} />
+        {!isEmptyObject(this.business.legal_address) && (
```

Comment:
I think it's a good call to create this `isEmptyObject` function and export it from the utils - but I think it makes this file a little less clean. 

Would you consider this as an alternative? 

What if we moved this logic inside the `DetailSection` component or inside the sub components? Since each of these `business-details` sub components are rendering `DetailSection` - maybe we do the check for `isEmptyObject` inside the detail components, such as inside `representative-details`, and then pass that boolean to `DetailSection` where it uses the boolean in a ternary to render the section or not. 

What do you think of this possibilty?

Reply:
Very good call!

I'll move this validation to inside each subcomponent 🙌